Makefile

```diff
@@ -25,7 +25,7 @@ BUILD_PATH := $(BASE_PATH)/build
 CMD_PATH := $(BASE_PATH)/cmd
 
 # Build information
-BUILD ?= $(shell date -Iseconds)
+BUILD ?= $(shell date +%FT%X%:z)
```

Comment:
Note that this format doesn't work with macOS:

```shell
$ date +%FT%X%:z
2019-01-31T10:50:56:z
```

Reply:
Is there something else that can produce `2019-01-31T20:36:34+02:00`?

I feel like I'll write a tiny Go program that will print the date in the right format.

Reply:
That seems like overkill.

Just leaving out the extra colon should get you most of the way there:

```
$ date +%FT%X%z
2019-01-31T10:59:58-0800
```

but note it omits the colons between the hours/minutes of the Z offset.

Reply:
It looks like the Linux `date` command supports `%z` without the colons for a similar effect, so if we don't care about the exact format but only consistency, that should probably work.

Reply:
Yeah, sorry for the delay, was working on the other issue. Pushed the `%z` and changed the parse format.